pages/common/pywal.md

```diff
@@ -0,0 +1,28 @@
+# pywal
```

Comment:
Perhaps we could rename this page to `wal` please? tldr pages should have a title of the command name they are about. We should rename the file too.

Reply:
Can do! I just wasn't sure which name to use, 'pywal' or, 'wal'.

Reply:
Yeah, we use the name of the command, because through the tldr CLI you're most likely to do `tldr {{command_name}}`.